libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -1108,6 +1143,18 @@ void NavEKF3::getVelNED(int8_t instance, Vector3f &vel) const
     }
 }
 
+// return estimate of true airspeed vector in body frame in m/s for the specified instance
+// An out of range instance (eg -1) returns data for the primary instance
+// returns false if estimate is unavailable
+bool NavEKF3::getAirSpdVec(int8_t instance, Vector3f &vel) const
+{
+    if (instance < 0 || instance >= num_cores) instance = primary;
```

Comment:
we wouldn't normally squeeze this all onto a single line,  maybe spread it out into 4 lines using curly brackets.